modules/access-control/src/access-control-constants.ts

```diff
@@ -132,6 +132,31 @@ export class AccessControlConstants {
      */
     public static readonly GROUP_DELETE: string = "group:delete";
 
+    /**
+     * Role feature permission
+     */
+    public static readonly ROLE: string = "role";
```

Comment:
WDYT about getting these through the provider. IMO, each app should provide these rather than defining in the common module.

Reply:
Yes, that will allow us to use the access control module in the My Account app as well. Since this is a change to the module, shall we create a separate issue for this?